Utilities/CodeGeneration/EdFi.Ods.CodeGen/Providers/Impl/AssemblyDataProvider.cs

```diff
@@ -70,6 +73,23 @@ public IEnumerable<AssemblyData> Get()
                 .Select(Create)
                 .ToList();
 
+            var extensionPaths = _extensionLocationPluginsProviderProvider.GetExtensionLocationPlugins();
+
+            foreach (var extensionPath in extensionPaths)
```

Comment:
can this be  linq statement? also we should have a test that validates this logic

Reply:
Added unit test method